packages/components/src/components/hds/accordion/index.ts

```diff
@@ -0,0 +1,16 @@
+import Component from '@glimmer/component';
+import type { ComponentLike } from '@glint/template';
+import type { HdsAccordionItemIndexSignature } from './item/index.ts';
+
+interface HdsAccordionIndexSignature {
+  Blocks: {
+    default: [
+      {
+        Item?: ComponentLike<HdsAccordionItemIndexSignature>;
```

Comment:
this makes me thing that we should provide some guidance around the `Blocks` in the signature, in the shared document https://github.com/hashicorp/design-system/pull/2087; until now I haven't paid much attention to this entity, because I thought the `npx` codemod would take care of this, but now I see here is much more structured than the ones I have added in my PRs. Do you think is it worth a quick 101 on how to do `Blocks` signatures?

Reply:
It depends very much on whether there is structured content (e.g. subcomponents) within the blocks

Reply:
We've discussed today, so I am closing this